great_expectations/datasource/data_connector/configured_asset_sql_data_connector.py

```diff
@@ -259,6 +301,88 @@ def build_batch_spec(
 
         return SqlAlchemyDatasourceBatchSpec(batch_spec)
 
+    def _get_sorters_from_splitter_method_name(
+        self, splitter_method_name: str, splitter_kwargs: Dict[str, Union[str, list]]
+    ) -> List[Sorter]:
+        """Given a splitter method and its kwargs, return an appropriately instantiated list of Sorters.
+
+        Args:
+            splitter_method_name: splitter name starting with or without preceding '_'.
+            splitter_kwargs: splitter kwargs dictionary for splitter directives.
+
+        Returns:
+            an ordered list of sorters required to sort splitter batches.
+        """
+        splitter_method_to_sorter_method_mapping: Dict[
+            str, Optional[Sorter]
+        ] = self.SPLITTER_METHOD_TO_SORTER_METHOD_MAPPING
+        splitter_method_name: str = self._get_splitter_method_name(
+            splitter_method_name=splitter_method_name,
+        )
+        try:
+            sorter_method: Optional[Sorter] = splitter_method_to_sorter_method_mapping[
+                splitter_method_name
+            ]
+        except KeyError:
+            raise ge_exceptions.SorterError(
+                f"No Sorter is defined in ConfiguredAssetSqlDataConnector.SPLITTER_METHOD_TO_SORTER_METHOD_MAPPING for splitter_method: {splitter_method_name}"
+            )
+
+        if sorter_method == DictionarySorter:
+            return [DictionarySorter(name=splitter_kwargs["column_name"])]
+        elif sorter_method == LexicographicSorter:
+            if splitter_method_name == "split_on_multi_column_values":
```

Comment:
Think the enum would provide a lot of value here

Reply:
I am doing this as part of this WIP PR: https://github.com/great-expectations/great_expectations/pull/5980